provider/pkg/schema/gen.go

```diff
@@ -936,6 +942,22 @@ func (ctx *context) propertySpec(propName, resourceTypeName string, spec *jssche
 		}
 	}
 
+	if propName == "Tags" {
```

Comment:
"Tags" is sure to always be in that casing?

Reply:
Actually, I think technically the name of the tags field is declared individually for each resource (eg. [here](https://github.com/pulumi/pulumi-aws-native/blob/96066277e4f414aade19612d31dc58fa78e91113/aws-cloudformation-schema/aws-autoscaling-autoscalinggroup.json#L6)) There are only a handful of resources that use something other than "Tags" though: 
```
rg tagProperty aws-cloudformation-schema | rg -v '/properties/Tags' | head
aws-cloudformation-schema/aws-datapipeline-pipeline.json:    "tagProperty" : "/properties/PipelineTags"
aws-cloudformation-schema/aws-cognito-userpool.json:    "tagProperty" : "/properties/UserPoolTags"
aws-cloudformation-schema/aws-backup-backupvault.json:    "tagProperty" : "/properties/BackupVaultTags"
aws-cloudformation-schema/aws-backup-reportplan.json:    "tagProperty" : "/properties/ReportPlanTags"
aws-cloudformation-schema/aws-efs-accesspoint.json:    "tagProperty" : "/properties/AccessPointTags"
aws-cloudformation-schema/aws-efs-filesystem.json:    "tagProperty" : "/properties/FileSystemTags"
aws-cloudformation-schema/aws-backup-backupplan.json:    "tagProperty" : "/properties/BackupPlanTags"
aws-cloudformation-schema/aws-backup-framework.json:    "tagProperty" : "/properties/FrameworkTags"
```

Reply:
And of course just for inconsistency, there are resources with a `Tags` field that don't  include the tagging metadata block. I feel like we would be more justified in skipping those ones though.